components/general/_generalpreview.hbs

```diff
@@ -6,7 +6,7 @@
     <title>{{ title }}</title>
 </head>
 <body>
-    {{{ inline "components/general/icons/icon-sprite.svg" }}}
+    {{{ inline "public/images/icons-sprite.svg" }}}
     {{{ yield }}}
 </body>
 </html>
```

Comment:
Remove this file and `components/blocks/_preview.hbs`, but add exactly same code as `components/_preview.hbs`.